src/main/kotlin/org/wfanet/measurement/kingdom/deploy/common/server/KingdomApiServer.kt

```diff
@@ -59,3 +61,29 @@ fun runKingdomApiServer(
 
   CommonServer.fromFlags(commonServerFlags, name, service).start().blockUntilShutdown()
 }
+
+fun runKingdomApiServer(
+  kingdomApiServerFlags: KingdomApiServerFlags,
+  duchyIdFlags: DuchyIdFlags,
+  commonServerFlags: CommonServer.Flags,
+  serviceFactory: (Channel) -> BindableService
+) {
+  DuchyIds.setDuchyIdsFromFlags(duchyIdFlags)
+
+  val serverCerts =
+    SigningCerts.fromPemFiles(
```

Comment:
I reused the `SigningCerts` struct and load it from command line flags that are passed to the server.

I think the bigger conversation we should have is, does this make sense (e.g. have the Kingdom API server parse and use flags instead of e.g. doing this in CommonServer (and by that, not repeat this in all the other servers).